crt_portal/cts_forms/validators.py

```diff
@@ -45,7 +45,7 @@ def _scan_file(file):
     try:
         # ClamAV only listens on 9443, which is SSL, but it self-signs its cert.
         # Because this request stays within the cloud.gov container, it's fine to not verify SSL.
-        return requests.post(settings.AV_SCAN_URL, files={'file': file}, data={'name': file.name}, timeout=25, verify=False)  # nosec
+        return requests.post(settings.AV_SCAN_URL, files={'file': file}, data={'name': file.name}, verify=False)  # nosec
```

Comment:
FYI:

this will now be limited by the cloud.gov request timeout - I'm not sure what this is (it's not immediately available in their docs) but most cloud services default to ~60 seconds

In local there will be no limit.

Reply:
> FYI:
> 
> this will now be limited by the cloud.gov request timeout - I'm not sure what this is (it's not immediately available in their docs) but most cloud services default to ~60 seconds
> 
> In local there will be no limit.

That is how it was before we added the timeout. Should be OK for now.